packages/devtools_app/lib/src/landing_screen.dart

```diff
@@ -67,6 +73,39 @@ class _LandingScreenBodyState extends State<LandingScreenBody> {
     );
   }
 
+  Widget _buildImportInstructions() {
+    return _buildSection(
+      title: 'Load DevTools Data',
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          _subtitleText(
+              'Import a data file to use DevTools without an app connection.'),
+          const SizedBox(height: denseRowSpacing),
+          _captionText(
+              'At this time, DevTools only supports importing files that were originally'
```

Comment:
What users is this caption for?

Reply:
This is to warn users about the files that we currently support. for example, we don't support loading an export from chrome://tracing or perfetto yet

Reply:
Maybe mention those specifically? You could even mention we plan to support them.